bears/hypertext/PHPCodeSnifferBear.py

```diff
@@ -0,0 +1,28 @@
+from coalib.bearlib.abstractions.Linter import linter
+from coalib.bears.requirements.DistributionRequirement import (
+    DistributionRequirement)
+
+
+@linter(executable='phpcs',
+        output_format='regex',
+        output_regex=r'(?P<filename>.+):(?P<line>\d+):(?P<column>\d+): '
+                      '(\w+) - (?P<message>.+)')
+class PHPCodeSnifferBear:
+    """
+    Tokenises and "sniffs" PHP, JavaScript and CSS files to detect violations
+    of a defined coding standard.
+
+    See <http://pear.php.net/manual/en/package.php.php-codesniffer.php> for
+    more information.
+    """
+
+    LANGUAGES = {"PHP", "JavaScript", "CSS"}
+    REQUIREMENTS = {DistributionRequirement(apt_get='phpcs')}
+    AUTHORS = {'The coala developers'}
+    AUTHORS_EMAILS = {'coala-devel@googlegroups.com'}
+    LICENSE = 'AGPL-3.0'
+    CAN_DETECT = {'Formatting', 'Syntax'}
```

Comment:
> t can also help prevent some common semantic errors made by developers.

I think it can do more, we should try to find out what it can do because I think it can do security relevant stuff which would be important